network.go

```diff
@@ -625,6 +626,58 @@ func (s *sandbox) updateRoute(netHandle *netlink.Handle, route *types.Route, add
 	return nil
 }
 
+// addARPNeighbors will take neighbors and add them to ARP entries.
+// This is used for mirror any static arp entries created by the network
+// plugin in the network namespace on the host side.
+func (s *sandbox) addARPNeighbors(netHandle *netlink.Handle, requestedNeighbors *pb.ARPNeighbors) error {
+	if requestedNeighbors == nil {
+		return errNoNeighbors
+	}
+
+	var err error
+	if netHandle == nil {
+		netHandle, err = netlink.NewHandle(unix.NETLINK_ROUTE)
+		if err != nil {
+			return err
+		}
+		defer netHandle.Delete()
+	}
+
+	for _, neighbor := range requestedNeighbors.ARPNeighbors {
+		// Find link index from route's device name.
+		link, err := netHandle.LinkByName(neighbor.Device)
+		if err != nil {
+			return grpcStatus.Errorf(codes.Internal, "Could not find link from device %s: %v", neighbor.Device, err)
+		}
+
+		toIP := net.ParseIP(neighbor.ToIPAddress.Address)
```

Comment:
`net.ParseIP()` returns `nil` on error so shouldn't we check that?

Reply:
My initial thought was netlink would error out in that case. I have added an explicit check for it now.